src/blocks/mrc_class_method_def.ts

```diff
@@ -458,7 +458,8 @@ export const pythonFromBlock = function (
         xfix2 = xfix1;
     }
     if(block.mrcPythonMethodName == '__init__'){
-        branch = generator.defineClassVariables() + branch;
+        branch = generator.INDENT + 'super().__init__()\n' +
```

Comment:
If the init is an opmode, then it needs to pass robot to the super().__init__

Reply:
This is assuming that the parent OpMode class should have the line:
`self.robot = robot` in it.